README.md

```diff
@@ -38,34 +38,43 @@ Jina documentation adheres to the following hierarchical structure. Each Jina pr
 
 If you find a gap in our documentation, please submit a GitHub issue [here](https://github.com/jina-ai/docs/issues/new).
 
-## How to add pages
+## Documentation Process
+
+All documentation should follow the same process as any other PR:
+
+1. Every developer who wrote the code should also write the Documentation.
+2. Documentation engineer will review the PR.
+3. After PR is approved by Doc-engineer it will be reviewed/edited by technical writer
```

Comment:
nit: After **the** PR is approved **by the** Doc-engineer it will be reviewed/edited by **a** technical writer.